test/data/instructions_with_path.yml

```diff
@@ -0,0 +1,14 @@
+---
+:class: ''
+:url: http://majles.marsad.tn/2014/fr/assemblee
+:path: members/id=212
+:to:
```

Comment:
what is `to` here? Is this meant to be `to_h`? If so, the tests not catching this error is quite a major warning sign…